packages/sources/iex-cloud/src/endpoint/stock.ts

```diff
@@ -34,6 +34,11 @@ export const execute: ExecuteWithConfig<Config> = async (request, _, config) =>
 
   const response = await Requester.request<ResponseSchema>(reqConfig)
   const result = Requester.validateResultNumber(response.data, ['latestPrice'])
+  const marketOpen = Requester.getMarketOpenFlag(response.data, ['isUSMarketOpen'])
```

Comment:
What if this was `validateResultBool` and took in arguments for the error messages?